community/kernel/src/main/java/org/neo4j/kernel/impl/store/MetaDataStore.java

```diff
@@ -808,7 +808,7 @@ public boolean closedTransactionIdIsOnParWithOpenedTransactionId()
         return onPar;
     }
 
-    public void logRecords( final Logger msgLog )
+    public void logRecords( final Consumer<String> consumer )
```

Comment:
Please remove Consumer from here it's over kill and useless complication that makes code less readable, if it still possible.
Instead if you think that getting logger is to verbose provide non verbose way of getting logger